ts/screens/wallet/ConfirmCardDetailsScreen.tsx

```diff
@@ -252,29 +258,41 @@ class ConfirmCardDetailsScreen extends React.Component<Props, State> {
             hideFavoriteIcon={true}
           />
           <View spacer={true} />
-          <NoticeBox
-            backgroundColor={customVariables.toastColor}
-            iconProps={{
-              name: "io-notice",
-              color: customVariables.brandDarkGray,
-              size: 24
+          <InfoBox
+            alignedCentral={true}
+            iconSize={24}
+            iconColor={IOColors.bluegreyDark}
+          >
+            <H5 weight={"Regular"}>{I18n.t("wallet.saveCard.notice")}</H5>
+          </InfoBox>
+          <View spacer large />
+          <View
+            style={{
+              flexDirection: "row",
+              justifyContent: "space-between"
             }}
+            testID={"test"}
           >
-            <Text>{I18n.t("wallet.saveCard.notice")}</Text>
-          </NoticeBox>
-          <View spacer={true} />
-          <Grid>
-            <Col size={5}>
-              <Text bold={true}>{I18n.t("wallet.saveCard.infoTitle")}</Text>
-              <Text>{I18n.t("wallet.saveCard.info")}</Text>
-            </Col>
-            <Col size={1}>
+            <View
+              style={{
+                ...IOStyles.flex
+              }}
+            >
+              <H4 weight={"SemiBold"} color={"bluegreyDark"}>
+                {I18n.t("wallet.saveCard.infoTitle")}
+              </H4>
+              <H5 weight={"Regular"} color={"bluegrey"}>
+                {I18n.t("wallet.saveCard.info")}
+              </H5>
+            </View>
+            <View hspacer={true} />
+            <View style={{ paddingTop: 7 }}>
               <Switch
                 value={this.state.setAsFavourite}
                 onValueChange={this.onSetFavouriteValueChange}
               />
-            </Col>
-          </Grid>
+            </View>
+          </View>
         </Content>
         {isInPayment ? (
```

Comment:
The design wants `TwoButtonsInlineThird` in place of `TwoButtonsInlineHalf` even when this screen is shown not in a payment flow

|  expected  | current |
| ------------- | ------------- |
![Preferences](https://user-images.githubusercontent.com/822471/151009734-5af04782-e38e-4f3c-b952-3a01e52b528a.png) | ![Privacy](https://user-images.githubusercontent.com/822471/151009731-bee092d0-4475-4545-a513-bd38132758c8.png)

Reply:
Thank you and sorry @thisisjp that already reported this error.
Updated in [01e90c0](https://github.com/pagopa/io-app/pull/3691/commits/01e90c050b83380fd139a48918806ce40277c17b)